### DISCUSSION TITLE
Display notice to user

### DISCUSSION THREAD

**Discussion by Jmac-71** [PSAppDeployToolkit/PSAppDeployToolkit#1100] [Q&A] [answered]
My office has an insane desire for doing silent installs and despite my best efforts to make leadership understand that is not always possible, since some apps need to be seen in order to lead to a graceful reboot.  The reboot issue has been resolved but an "unannounced" reboot is problematic.

With this information in mind an item I have been fighting with in versions 3.6, 3.8, and 3.9 is a means to display a notice at the end of a silent deployment notifying the user of next actions.  For example, a simple message please reboot sometime today or give them the option to reboot now or delay for x hours.  I have tried various suggestions I found the PSADT forums and other net sources with no success.  

Any help would be appreciated and if this not an option with the tool in its current state without heroic action take this post as an enhancement request for future releases.

**Comment by PZan** [comment]
What tool do you use for deploying your applications? Both Intune and ConfigMgr has various settings for restart behavior, which should be sufficient for your org needs.

**Comment by mjr4077au** [comment, marked as answer]
What you could do here in v4 is after the call to `Close-ADTSession`, this is where you could place your call to `Show-ADTInstallationRestartPrompt`. This way, the deployment could be silent, but the call to `Show-ADTInstallationRestartPrompt` will still show as the session is closed and therefore it has no awareness that of whether its silent or not.

Feel free to test it out and let us know how you go!

**Comment by Jmac-71** [comment]
We are using both SCCM and Intune with the intention of using Intune more often


==================================
Jonathan McGrail
Endpoint Engineering Team
***@***.******@***.***>
26700 Lahser Road, 4th Floor
Southfield, MI 48076
Mailcode: LH04
tel: (248) 395-5196

Do you need to open a case?:
Use our Self-Service Portal at: https://mclarenhealth.service-now.com/ssp

From: Pelle ***@***.***>
Sent: Thursday, December 5, 2024 1:42 AM
To: PSAppDeployToolkit/PSAppDeployToolkit ***@***.***>
Cc: McGrail, Jonathan ***@***.***>; Author ***@***.***>
Subject: [EXTERNAL] Re: [PSAppDeployToolkit/PSAppDeployToolkit] Display notice to user (Discussion #1100)

WARNING: EXTERNAL EMAIL. PLEASE EXERCISE CAUTION WHEN CLICKING ON HYPERLINKS.
This Message originated outside your organization.
________________________________
What tool do you use for deploying your applications? Both Intune and ConfigMgr has various settings for restart behavior, which should be sufficient for your org needs.

—
Reply to this email directly, view it on GitHub<https://github.com/PSAppDeployToolkit/PSAppDeployToolkit/discussions/1100#discussioncomment-11469377>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BNOS62AWO36IJP6KK5STALT2D7YU7AVCNFSM6AAAAABTBAN5LOVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCNBWHEZTONY>.
You are receiving this because you authored the thread.Message ID: ***@***.***>
McLaren confidentiality statement: "The information contained in this communication, including attachments, is confidential, may be privileged, and is intended only for the use of the named recipient(s). Unauthorized use, disclosure, forwarding or copying is strictly prohibited and may be unlawful. If you have received this communication in error, please notify me IMMEDIATELY at the phone number or pager listed above."
